src/frontend/ts/state/chessStore.ts

```diff
@@ -112,6 +114,58 @@ const useChessStore = create<ChessState>((set, get) => ({
     await get().fetchArchivedMoves(gameID)
     get().setDisplayGame(get().archivedGames.get(gameID))
   },
+  countTallies: async (archivedGames: Map<GameID, ArchivedGameInfo>) => {
+    const tallies = get().tallies
+    // initialize empty record of players' wins, losses, and draws
+    const winsLossesDraws: Map<Ship, { wins: number, losses: number, draws: number }> = new Map();
```

Comment:
`{ wins: number, losses: number, draws: number }` should have its own type (interface) in a new `chess.ts` file in `ts/types/`:
```
export interface Record {
  wins: number,
  losses: number,
  draws: number
}
```